examples/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/README.md

```diff
@@ -0,0 +1,13 @@
+## NeurIPS2020 L2RPN Challenge
+
+The **PARL** team gets the first place for both tracks (Robustness Track and Adaptability Track) in *NeurIPS2020 Learning-to-Run-a-Power-Network* challenge! 
+
+<p align="center">
+<img src="images/l2rpn.jpeg" alt="PARL" height="300" />
+</p>
+
+## Codes for evaluation
```

Comment:
Note: We release the code with trained policy for our top submission at the moment.  Regarding the training code, we are still working on a technical report and will release it as soon as possible.

Reply:
updated